test/util/all_close_f.cpp

```diff
@@ -127,6 +191,50 @@ uint32_t test::matching_mantissa_bits(uint32_t distance)
     return matching_matissa_bits;
 }
 
+uint64_t test::matching_mantissa_bits(uint64_t distance)
+{
+    uint64_t tolerance_needed = distance;
+
+    if (tolerance_needed < 0x8000000000000000)
```

Comment:
This can be simplified in another PR as I discussed with @gcwenger .

Reply:
+1